src/foam/comics/DAOController.js

```diff
@@ -186,7 +186,7 @@ foam.CLASS({
         var sectionName = undefined;
         var hasManySections = false;
         var model = this.data.of.model_;
-        while ( model && model !== foam.core.FObject ) {
+        while ( model && model === foam.core.FObject ) {
```

Comment:
@lchanmann does this make sense to you? Why was it `!==` originally? Why can't we just do `while ( model ) {` ?

Reply:
Line 201, we look up the parent of the model then call `.model_`. I think we can compare `model && model.id !== 'foam.core.FObject'` instead.

The logic here is to traverse the model ancestors until we can determine if the model has many sections. And FObject is the root all models so we want to stop there.

Reply:
This doesn't make sense to me. It looks like if it is any model other than FObject it will just not work. Also, easier to traverse up the classes by starting with this.data.of and then keep calling getSuperClass(). Also, 'model' is a Model but foam.core.FObject is a Class, which is probably the problem. Then from the class get the sections as cls.model_.sections.